src/app/shared/ui-kit/ui-question-card/ui-question-card.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="w-80 space-y-10 rounded-xl bg-success p-6 shadow-xl md:w-[640px]">
+  <div class="flex items-center justify-between">
+    <p>
+      <span class="bg-warning p-1 text-3xl font-semibold text-primary">1</span>
+      <span class="p-1 font-medium">/10</span>
+    </p>
+    <img class="h-20 w-20" [src]="avatarPath" alt="category icon" />
+  </div>
+  <h5 class="text-2xl-plus font-semibold md:text-3xl">
+    Let's begin with this
+    <span class="bg-error p-1">legendary pop group</span>
+    <p>What's their name?</p>
+  </h5>
+
+  <form class="flex flex-col items-center">
+    @for (option of mockOptions; track option) {
+      <quiz-ui-radio-group
```

Comment:
Could you tell me why we are iterating the radio options here instead of radio group component?

Reply:
Because the checked style didn't work correctly when I added the formControl as an input

Reply:
I checked again. The problem wasn't in this part. So, I reverted the iteration back to the radio-group component.